code/_globalvars/misc.dm

```diff
@@ -41,3 +41,8 @@ GLOBAL_LIST_EMPTY(topic_tokens)
 GLOBAL_PROTECT(topic_tokens)
 GLOBAL_LIST_EMPTY(topic_servers)
 GLOBAL_PROTECT(topic_servers)
+
+// wizard stuff
+GLOBAL_VAR_INIT(magical_access, FALSE)
+GLOBAL_VAR_INIT(magical_access_recursive, FALSE)
+GLOBAL_VAR_INIT(same_intent_recursive, FALSE)
```

Comment:
I hate how this is implemented

Reply:
I can remove recursive globals, but I wanted a possible way that an admin can stop this.

Reply:
but also, `GLOBAL_VAR_INIT(magical_access, FALSE)` was the easiest way to handle this because the first intention of this PR was to remove all access requirements of airlocks, but it's weirdly recovering its access requirement. I couldn't find any reason why it does so, so, magical_access was the solution to let people bypass things without id card.

Reply:
It has only `magical_access` global now, and the ritual has static variable for recursive.

Reply:
Okay, I really didn't need this. I used `var/static/magical_access` in /obj types.